transfocate/tests/__init__.py

```diff
@@ -3,6 +3,6 @@
 from . import test_transfocate
 
 # Create a Transfocator with a few inserted lenses
-transfocator = test_transfocate.transfocator()
-test_transfocate.insert(transfocator.prefocus_mid)
-test_transfocate.insert(transfocator.tfs_02)
+#transfocator = test_transfocate.transfocator()
```

Comment:
I think the docs need this. Maybe just move the fake transfocator into a non-fixture function then call it in a fixture.

```python
def sim_transfocator():
    make_fake_class(Transfocator)
    ...

@pytest.fixture(scope='function')
def transfocator():
     return sim_transfocator()
```